datadog_checks_base/datadog_checks/base/checks/openmetrics/mixins.py

```diff
@@ -917,7 +927,8 @@ def _submit_gauges_from_histogram(self, metric_name, metric, scraper_config, hos
             if self._ignore_metrics_by_label(scraper_config, metric_name, sample):
                 continue
             custom_hostname = self._get_hostname(hostname, sample, scraper_config)
-            if sample[self.SAMPLE_NAME].endswith("_sum") and not scraper_config['send_distribution_buckets']:
+            if sample[self.SAMPLE_NAME].endswith("_sum") and \
+                    (not scraper_config['send_distribution_buckets'] or scraper_config['send_distribution_sums']):
```

Comment:
Can you make it more clear your logic for `not scraper_config['send_distribution_buckets'] or scraper_config['send_distribution_sums']`? It can be a little confusing if you mean:
1. `(not scraper_config['send_distribution_buckets']) or scraper_config['send_distribution_sums']`
2. `not (scraper_config['send_distribution_buckets'] or scraper_config['send_distribution_sums'])`